src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -588,5 +577,52 @@ private static bool AcceptedTokenVersionMismatch(MsalUiRequiredException msalSev
             // This is subject to change in the future
             return msalSeviceException.Message.Contains(ErrorCodes.B2CPasswordResetErrorCode, StringComparison.InvariantCulture);
         }
+
+        private async Task<ClaimsPrincipal?> GetAuthenticatedUserAsync(ClaimsPrincipal? user)
+        {
+            if (user == null && _httpContextAccessor.HttpContext != null)
```

Comment:
```suggestion
            if (user == null && _httpContextAccessor.HttpContext?.User != null)
```

Reply:
got it. thanks!

---
In reply to: [464811695](https://github.com/AzureAD/microsoft-identity-web/pull/383#discussion_r464811695) [](ancestors = 464811695)